app/models/grda_warehouse/hud/client.rb

```diff
@@ -754,26 +754,6 @@ def alternate_names
       names.join(',')
     end
 
-    def client_names(user:, health: false)
```

Comment:
moved to SourceClientNameSet. Note, afaict, the 'health' flag is always true given current usage. I removed it as a simplification